### ISSUE TITLE
blackout and range mode

### ISSUE THREAD

**Issue by Valantir007** [ChiperSoft/Kalendae#201]
If mode is range and blackout is set, I can range date which is in blackout array.. :/

**Comment by ChiperSoft** [comment]
Yes, the library only supports blackouts for controlling date selection, it doesn't enforce range finding.
